src/main/scala/io/iohk/ethereum/utils/Config.scala

```diff
@@ -22,6 +22,8 @@ object Config {
 
   val keysFile: String = config.getString("keys-file")
 
+  val shutdownTimeout: Duration = config.getDuration("shutdown-timeout").toMillis.millis
```

Comment:
Previously we were not shutting down the actorsystem on exit, so once that was added a 'wait' for the shutdown to complete was necessary and so a `shutdownTimeout` was born.